input/core/src/commonMain/kotlin/majestic/RadioButton.kt

```diff
@@ -0,0 +1,97 @@
+package majestic
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.CheckCircle
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.PointerIcon
+import androidx.compose.ui.input.pointer.pointerHoverIcon
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun RadioButton(
+    modifier: Modifier = Modifier,
+    selected: Boolean,
+    enabled: Boolean = true,
+    label: String,
+    selectedColor: Color = Color.Unspecified,
```

Comment:
Can we add a class to minimize the number of parameters to this Composables??

intead of it having all these colors as seperate paraameters, we can have something like

```kotlin
data class RadioColorGroup(
  val background: Color,
  val label: Color,
  val border: Color
) {
  companion object {
      val default by lazy {
        RadioColorGroup(Color.Unspecified,Color.Unspecified,Color.Unspecified)
      }
   }
}

data class RadioColors(
  val selected: RadioColorGroup = RadioColorGroup.Default,
  val unselected: RadioColorGroup = RadioColorGroup.Default,
  val disabled: RadioColorGroup = RadioColorGroup.Default
)
```

after having such classes, we can now go ahead and change the composable from
```kotlin
fun RadioButton(
    ...
    selectedLabelColor: Color = Color.Unspecified,
    selectedBorderColor: Color = Color.Unspecified,
    unselectedColor: Color = Color.Unspecified,
    unselectedLabelColor: Color = Color.Unspecified,
    unselectedBorderColor: Color = Color.Unspecified,
    disabledColor: Color = Color.Unspecified,
    disabledLabelColor: Color = Color.Unspecified,
    disabledBorderColor: Color = Color.Unspecified,
    onSelect: () -> Unit
)
```

into


```kotlin
fun RadioButton(
    ...
    colors: RadioColors = RadioColors(),
    onSelect: () -> Unit
)
```